src/main/java/uk/gov/dvsa/enums/DocumentsConfig.java

```diff
@@ -50,7 +50,10 @@ public enum DocumentsConfig {
             new String[]{CertificateTemplates.VTP20W.getCertificateTemplateName()}),
     VTP30(CertificateTypes.CVS_FAIL.getCertificateType(), uk.gov.dvsa.model.cvs.VTP30.class,
             new String[]{CertificateTemplates.VTP30.getCertificateTemplateName()}),
-
+    VTP30W(CertificateTypes.CVS_FAIL_WELSH.getCertificateType(), uk.gov.dvsa.model.cvs.VTP30W.class,
+            new String[]{CertificateTemplates.VTP30W.getCertificateTemplateName()}),
+    VTP30_FAIL_BILINGUAL(CertificateTypes.CVS_FAIL_WELSH_BILINGUAL.getCertificateType(), VTP30Bilingual.class,
```

Comment:
Same here, wonder if it's worth refactoring to CVS_FAIL_BILINGUAL for consistency as it's not really a VTP30 at this point, it's both a VTP30 and VTP30W.